src/features/jobTracker/ui/NewJobDialog/NewJobDialog.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import {
+  Dialog,
+  Modal,
+  ModalOverlayProps,
+  ModalOverlay,
+} from 'react-aria-components'
+import { useRouter } from 'next/navigation'
+import { JOB_TRACKER_BASEURL } from '../../data/contants/routes'
+import { NewJobForm } from '../NewJobForm'
+
+export function NewJobDialog(props: ModalOverlayProps) {
+  const router = useRouter()
+
+  const handleOpenChange = () => {
+    router.push(JOB_TRACKER_BASEURL)
+  }
+
+  return (
+    <ModalOverlay
```

Comment:
NOTE: This is duplicated Modal markup and it can be extracted to a new component, I was thinking doing so perhaps naming it ModalWindow.

Reply:
sounds good